client/my-sites/upgrades/domain-management/edit/mapped-domain.jsx

```diff
@@ -117,4 +117,5 @@ const MappedDomain = React.createClass( {
 	}
 } );
 
+export { MappedDomain };
```

Comment:
I didn't do it above because the `linter` would start crying. :sob: If I remove the brackets `linter` cries again. :sob: